README.md

```diff
@@ -24,3 +24,13 @@ This plugin provides hooks:
 This plugin allows you to store configuration files which reside in the dokku app directory, such as ENV, inside the app's git repository.
 
 For example, say you want to include a file called DOCKER_OPTIONS for the [Docker options plugin](https://github.com/dyson/dokku-docker-options). In the root directory of your app repository, create a directory called `.dokku` and put the files in there. This plugin will copy those files into `$DOKKU_ROOT/$APP/` before building the app.
+
+Optionally if you place a `crontab` in `.dokku` it will move this crontab file to `/etc/cron.d/$APP`
+
+You will also need to edit sudoers to add some more permissions to the dokku user.
+
+Add the following line. It will allow the dokku user to overwrite the old crontab as well as restart the cron service after it has been overwritten.
+
+```bash
+dokku ALL=(ALL) NOPASSWD: /bin/cp, /usr/sbin/service cron restart
```

Comment:
This should just be added to it's own `sudoers` file during the `install` phase of the plugin.